### PR TITLE
Feat(server logs format) - change logs format

### PR DESCRIPTION
**Pull request by nevobonneh**

### CODE REVIEW

**Review by NaamanTsur — APPROVED** [review]

File: server/src/ClientToDBAPI/ContactedPeople/mainRoute.ts [hunk 1]
```diff
@@ -143,50 +88,36 @@ ContactedPeopleRoute.post('/interactedContacts', (request: Request, response: Re
...
         })
         .catch(error => {
-            logger.error({
-                service: Service.SERVER,
-                severity: Severity.LOW,
-                workflow,
-                step: `got error from DB from graphql API: ${error}`,
-                user: response.locals.user.id,investigation: response.locals.epidemiologynumber
-            });
+            interactedContactsLogger.error(`got error from DB from graphql API: ${error}`, Severity.HIGH);
```
Comment (doronlnd84) [inline server/src/ClientToDBAPI/ContactedPeople/mainRoute.ts:108]: Did you mean Severity.Low?

File: server/src/ClientToDBAPI/ExposureRoute/mainRoute.ts [hunk 2]
```diff
@@ -50,35 +50,19 @@ const convertExposuresFromDB = (result: ExposureByInvestigationId) : Exposure[]
...
         .catch(err => {
-            logger.error({
-                service: Service.SERVER,
-                severity: Severity.LOW,
-                workflow: 'Getting exposures',
-                step: `got errors approaching the graphql API ${err}`,
-                investigation: response.locals.epidemiologynumber,
-                user: response.locals.user.id
-            });
+            exposuresLogger.error(`got errors approaching the graphql API ${err}`, Severity.HIGH);
```
Comment (doronlnd84) [inline server/src/ClientToDBAPI/ExposureRoute/mainRoute.ts:65]: did you mean Severity.LOW,?

File: server/src/ClientToDBAPI/IntersectionsRoute/mainRoute.ts [hunk 3]
```diff
@@ -26,24 +26,15 @@ const duplicatesErrorMsg = 'found duplicate ids';
...
     } else {
-        logger.error({
-            service: Service.SERVER,
-            severity: Severity.LOW,
-            workflow,
-            step: `got errors approaching the graphql API ${errorMsg}`,
-            investigation: response.locals.epidemiologynumber,
-            user: response.locals.user.id
-        });
+        handleDBErrorsLogger.error(`got errors approaching the graphql API ${errorMsg}`, Severity.HIGH);
```
Comment (doronlnd84) [inline server/src/ClientToDBAPI/IntersectionsRoute/mainRoute.ts:37]: LOW?

File: server/src/ClientToDBAPI/IntersectionsRoute/mainRoute.ts [hunk 4]
```diff
@@ -53,39 +44,23 @@ intersectionsRoute.get('/', (request: Request, response: Response) => {
...
-        logger.error({
-            service: Service.SERVER,
-            severity: Severity.LOW,
-            workflow: 'Getting Places And SubTypes By Types',
-            step: `got errors approaching the graphql API ${err}`,
-            investigation: response.locals.epidemiologynumber,
-            user: response.locals.user.id
-        });
-        response.status(errorStatusCode).send(err);
+            getPlacesSubTypesByTypesLogger.error(`got errors approaching the graphql API ${err}`, Severity.HIGH);
```
Comment (doronlnd84) [inline server/src/ClientToDBAPI/IntersectionsRoute/mainRoute.ts:62]: LOW?

File: server/src/ClientToDBAPI/ContactedPeople/mainRoute.ts (hunk 1 above)
Comment (NaamanTsur) [reply server/src/ClientToDBAPI/ContactedPeople/mainRoute.ts:108]: I doubt the original Severity.LOW was correct, considering that in every other catch occasion we use Severity.HIGH. It's probably an mistake @nevobonneh fixed.

**Review by doronlnd84 — APPROVED** [review]

**Pull request merged**